src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Linux.cs

```diff
@@ -230,6 +233,10 @@ private sealed class RunningInstance
             /// Maximum length of a name returned from inotify event.
             /// </summary>
             private const int NAME_MAX = 255; // from limits.h
+            /// <summary>
+            /// Flags that signify that we need to add a new directory to the watch.
+            /// </summary>
+            private const Interop.libc.NotifyEvents AddMaskFilters = Interop.libc.NotifyEvents.IN_CREATE | Interop.libc.NotifyEvents.IN_MOVED_TO;
```

Comment:
Does this need to be a const on the type like this, or can it be a local const as is done elsewhere in the method, e.g. just before it's used to set addWatch?